src/workerd/io/compatibility-date.capnp

```diff
@@ -947,4 +947,10 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # This flag is intended to be a roll-up flag. That is, as additional APIs
   # reach EOL, new compat flags will be added for those that will have
   # `impliedByAfterDate(name = "removeNodeJsCompatEOL", ...` annotations.
+
+  transpileTypescript @109 :Bool
+    $compatEnableFlag("transpile_typescript")
+    $compatDisableFlag("no_transpile_typescript")
+    $experimental;
+  # Transpile all loaded files to remove Typescript types.
```

Comment:
I would call this `typescriptErasableSyntax`, which is the more common, familiar name to folks who use this.

```suggestion
  typescriptErasableSyntax @109 :Bool
    $compatEnableFlag("typescript_erasable_syntax")
    $experimental;
  # Support typescript erasable syntax support.
  # This flag is expected to always be opt-in and never have
  # a default on date.
```

Reply:
it is now called `typescript_strip_types` which is even more to the point, wdyt?